File: CS/Helpers/DragDrop/ListBoxControlHelper.cs
using DevExpress.XtraEditors;
using DevExpress.XtraGrid.Columns;
using System;
using System.Collections.Generic;
using System.Linq;

namespace dxQ529392 {
    public class ListBoxControlHelper : BaseDropGridColumnHeaderSupportHelper<ListBoxControl> {
        public override void DropObjectCore(string columnFieldName, System.Drawing.Point location, ListBoxControl control) {
            control.Items.Add(columnFieldName);
        }
    }
}

File: VB/Helpers/DragDrop/ListBoxControlHelper.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraEditors
Imports DevExpress.XtraGrid.Columns
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace dxQ529392
	Public Class ListBoxControlHelper
		Inherits BaseDropGridColumnHeaderSupportHelper(Of ListBoxControl)
		Public Overrides Sub DropObjectCore(ByVal columnFieldName As String, ByVal location As System.Drawing.Point, ByVal control As ListBoxControl)
			control.Items.Add(columnFieldName)
		End Sub
	End Class
End Namespace
